dotCMS/src/integration-test/java/com/dotcms/publishing/PublisherFilterImplTest.java

```diff
@@ -0,0 +1,146 @@
+package com.dotcms.publishing;
+
+import com.dotcms.publisher.util.PusheableAsset;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.util.UUIDGenerator;
+import java.util.ArrayList;
+import java.util.Arrays;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class PublisherFilterImplTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5458951)